src/main/java/com/tasker/server/rest/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,31 @@
+package com.tasker.server.rest;
+
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.FieldError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@RestControllerAdvice
+public class GlobalExceptionHandler extends ResponseEntityExceptionHandler {
```

Comment:
given that GlobalExceptionHandler class is under this path src/main/java/com/tasker/server/rest, will it be available for boards rest controllers which is under this path ? src/main/java/com/tasker/server/boards/rest/

Reply:
Yes